recipes/requests-kerberos/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "requests-kerberos" %}
+{% set version = "0.12.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "9d21f15241c53c2ad47e813138b9aee4b9acdd04b82048c4388ade15f40a52fd"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
```

Comment:
Is `setuptools` really needed here? Usually `pip` and `python` are enough for the `host` section.

Reply:
`setuptools` is required for the build of this package, so I would rather list is explicitly than rely upon the fact that `pip` happens to also depend upon `setuptools`.